README.md

```diff
@@ -34,6 +34,13 @@ You can add as many aliases as you want in the corresponding configuration file
 Each alias is a key/value combination:
 > daily_alias = BSDEV-42
 
+### Internal issue references
+
+Internal issue prefixes are used to avoid adding 'to dispatch' time on internal entries
+
+You can add as many internal issue prefixes as you want in the corresponding configuration file section.
+> internal_issue_prefixes = BS-, BSRD-, ABCH-, ABFR-
```

Comment:
dispatch_blacklist_prefixes

Reply:
non_invoiceable_card_prefixes ?

Reply:
the real purpose is to blacklist some issues you don't want to dispatch in (invoiceable concept is not a gtimelog2odoo one in fact, just te result of the operation until you send your timesheet)